src/plotIt.cc

```diff
@@ -943,7 +951,7 @@ namespace plotIt {
     // Build legend
     TLegend legend(legend_position.x1, legend_position.y1, legend_position.x2, legend_position.y2);
     legend.SetTextFont(43);
-    legend.SetFillStyle(0);
+    legend.SetFillStyle(4000);
```

Comment:
This set the legend to be always transparent? What's the difference with respect to `0`?

Reply:
It seems to work with `0`, no idea why I changed it at that time... I'll revert that.